### PR TITLE
chore(gatsby-plugin-clerk): Add transition period notice

### PR DESCRIPTION
**Pull request by wobsoriano**
## Description

This PR adds notice of plan to EOL Gatsby SDK

Fixes ECO-86 and ECO-92

## Checklist

- [ ] `npm test` runs as expected.
- [ ] `npm run build` runs as expected.
- [ ] (If applicable) [JSDoc comments](https://jsdoc.app/about-getting-started.html) have been added or updated for any package exports
- [ ] (If applicable) [Documentation](https://github.com/clerk/clerk-docs) has been updated

## Type of change

- [ ] 🐛 Bug fix
- [ ] 🌟 New feature
- [ ] 🔨 Breaking change
- [x] 📖 Refactoring / dependency upgrade / documentation
- [ ] other:

### CODE REVIEW

File: .changeset/config.json [hunk 1]
```diff
@@ -12,7 +12,7 @@
   "access": "public",
   "baseBranch": "origin/main",
   "updateInternalDependencies": "patch",
-  "ignore": ["gatsby-plugin-clerk"],
+  "ignore": [],
```
Comment (wobsoriano) [inline .changeset/config.json:15]: Temporarily removing the gatsby SDK to ignored list so we can publish the notice to NPM

File: packages/gatsby-plugin-clerk/README.md [hunk 2]
```diff
@@ -28,6 +28,9 @@
 
 ---
 
+> [!IMPORTANT]
+> Starting August 1, 2024, the Gatsby SDK is entering a one-month notice period. We are actively seeking a new community maintainer. For full details, please see our [changelog](https://clerk-git-rob-eco-93-changelog-entry-about-the-start-of-f25dc8.clerkstage.dev/changelog/2024-08-01-gatsby-eol).
```
Comment (wobsoriano) [inline packages/gatsby-plugin-clerk/README.md:32]: Update this with the correct  link when changelog is published

**Review by nikosdouvlis — APPROVED** [review]
LGTM 👍🏻 
Let's make sure that this package gets released under the `4` version tag before release

**Comment by wobsoriano** [comment]
@nikosdouvlis thanks for reviewing!

That just means updating the version in the package.json file back to 4.4.7 (the last main version), right? Or are there any other steps required?

**Review by kylemac — APPROVED** [review]

**Pull request merged**